src/borg/archiver.py

```diff
@@ -995,8 +1000,8 @@ def _delete_archives(self, args, repository):
         """Delete archives"""
         manifest, key = Manifest.load(repository, (Manifest.Operation.DELETE,))
 
-        if args.location.archive:
-            archive_names = (args.location.archive,)
+        if args.location.archive or args.archives:
+            archive_names = tuple([args.location.archive] + args.archives )
```

Comment:
pep8, space before ).

Reply:
also, for the args.archives case of the "or", args.location.archive will be None (or '')?